client/layout/logged-out.jsx

```diff
@@ -32,17 +32,14 @@ const LayoutLoggedOut = ( {
 	let masterbar = null;
 
 	if ( useOAuth2Layout ) {
-		const hasValidOAuth2ClientData = !! oauth2Client;
-		const oauthClientName = hasValidOAuth2ClientData && oauth2Client.name;
-		classNameObject.dops = hasValidOAuth2ClientData;
-		classNameObject[ oauthClientName ] = hasValidOAuth2ClientData;
-
-		if ( oauthClientName ) {
-			masterbar = <OauthClientMasterbar oauth2Client={ oauth2Client } />;
+		// Uses custom styles for DOPS clients - which are the only ones with the name property defined
+		if ( oauth2Client.name ) {
```

Comment:
Would we want to use the conditional `( oauth2Client && oauth2Client.name )` here? The original logic accounted for `oauth2Client` being falsey (undefined, null, etc.).

Reply:
No need, `useOAuth2Layout` is set by the `showOAuth2Layout()` selector, which does this test.